From 5d61d46ebde65ed37c5c7f11925f11438c55dc27 Mon Sep 17 00:00:00 2001
From: Paul Gideon Dann <pdgiddie@gmail.com>
Date: Tue, 20 Jun 2017 09:59:20 +0100
Subject: [PATCH 4/5] Don't built qt4 and qt5 tests and demos in CMake

Hopefully they'll be fixed by the next Poppler release.
---
 qt4/CMakeLists.txt | 4 ++--
 qt5/CMakeLists.txt | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/qt4/CMakeLists.txt b/qt4/CMakeLists.txt
index 4d345681..567ff26e 100644
--- a/qt4/CMakeLists.txt
+++ b/qt4/CMakeLists.txt
@@ -2,5 +2,5 @@
 string(REPLACE "-Wsuggest-override" " " CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
 
 add_subdirectory(src)
-add_subdirectory(tests)
-add_subdirectory(demos)
+# add_subdirectory(tests)
+# add_subdirectory(demos)
diff --git a/qt5/CMakeLists.txt b/qt5/CMakeLists.txt
index 6bd7a6be..d7f09a72 100644
--- a/qt5/CMakeLists.txt
+++ b/qt5/CMakeLists.txt
@@ -1,5 +1,5 @@
 set(CMAKE_AUTOMOC ON)
 
 add_subdirectory(src)
-add_subdirectory(tests)
-add_subdirectory(demos)
+# add_subdirectory(tests)
+# add_subdirectory(demos)
-- 
2.13.1

